Misc/NEWS.d/next/Library/2022-02-01-11-32-47.bpo-46581.t7Zw65.rst

```diff
@@ -0,0 +1,3 @@
+Propagates :attribute:`~typing._GenericAlias._typevar_types` and
```

Comment:
All this are implementation details. A common user knows nothing about it. Describe how the bug affected a common user.

Reply:
Updated: https://github.com/python/cpython/pull/31061/files#diff-342ce6eb72310ee05e41d8897f6726904fd2bcbe83c79cf4f58e344e366db1c4.